src/main/java/build/buildfarm/worker/cgroup/Cpu.java

```diff
@@ -22,32 +22,50 @@ public class Cpu extends Controller {
   }
 
   @Override
-  public String getName() {
+  public String getControllerName() {
     return "cpu";
   }
 
+  @Deprecated(forRemoval = true)
   public int getShares() throws IOException {
     open();
     return readLong("cpu.shares");
   }
 
+  /**
+   * Represents how much CPU shares are allocated. Note - this method does nothing for CGroups v2.
+   * All processes get equal weight.
```

Comment:
This will not work - if we cannot allot more shares for a single process, then we can't have one execution able to take up multiple cores upon request.